build.xml

```diff
@@ -35,13 +35,32 @@
 
   <target name="codestyle" depends="prepare">
     <phpcodesniffer standard="codingstyle.xml">
+      <formatter type="full" usefile="false"/>
       <formatter type="checkstyle" outfile="${builddir}/checkstyle.xml"/>
-      <fileset dir="features">
+      <fileset dir="centreon-build">
```

Comment:
Unless I am mistaken, the _centreon-build_ directory should never be included in Centreon Web's reports.